views/login.hbs

```diff
@@ -0,0 +1,26 @@
+<div class="row">
+  <form class="col s12" action="/login" method="post">
+    {{# if user.errors}}
+      <div class="row">
+        <div id="loginFormError" class="col s12 pink-text">
+          {{user.errors}}
+        </div>
+      </div>
+    {{/if}}
+    <div class="row">
+      <div class="input-field col s12">
+        <input id="email" name="email" type="email" class="validate">
+        <label for="email">Email</label>
+      </div>
+    </div>
+    <div class="row">
+      <div class="input-field col s12">
+        <input id="password" name="password" type="password" class="validate">
```

Comment:
what does validate mean here from materialize?

Reply:
It turns it green after input... ech, it's fine. We have automatic email FE "validation" which is nice, and turns the input field green, so why not keep email and pw behaviour similarish.